src/pages/General/Home.tsx

```diff
@@ -1,20 +1,208 @@
 import React from 'react';
 import About from './About';
 import Services from './Services';
```

Comment:
As these imports, (About and Services) are never used, it is failing the CI test. This can be checked by visiting the "Actions" tab on github and navigating to your cuurent Pull-request. Let me know if you are stuck
![image](https://user-images.githubusercontent.com/59163024/167325112-7c3bf00f-ec2e-44d8-b8a4-86253cfc3315.png)